src/features/defi/providers/idle/components/IdleManager/Overview/IdleOverview.tsx

```diff
@@ -46,6 +49,27 @@ const defaultMenu: DefiButtonProps[] = [
   },
 ]
 
+export type OpportunityById = typeof BASE_OPPORTUNITIES_BY_ID
```

Comment:
erroneous addition? doesn't seem to be consumed anywhere?

Reply:
This must of been part of our crazy typescript stuff we were doing to make this more intelligent. I'll remove.